src/esp/gfx_batch/Renderer.h

```diff
@@ -81,6 +81,14 @@ struct RendererConfiguration {
   RendererConfiguration& setTileSizeCount(const Magnum::Vector2i& tileSize,
                                           const Magnum::Vector2i& tileCount);
 
+  /**
+   * @brief Set max light count per draw
+   *
+   * By default no lights are used, i.e. flat-shaded rendering.
+   * @see @ref Renderer::maxLightCount()
+   */
+  RendererConfiguration& setMaxLightCount(Magnum::UnsignedInt count);
```

Comment:
How about a comment with some guidance for how to pick this number? E.g. Assuming a single draw can possibly be the entire scene, then the max count should probably be similar to the expected max light count across all scenes (is that even true?).

Reply:
It's the max number of lights applied per draw.

The semantic will change slightly once light culling is in (for which it's essential that the lights contain range info, which they do in glTF files but not yet in Habitat-supplied lights). With light culling, this will be an upper bound of lights used per draw (picking the brightest from the set, ignoring the rest) and the number of lights in total will be handled internally, without any need for user-specified settings. 

Currently without the culling it's both the max number of lights applied per draw and the max number in total, because that's the same thing. I'll update the docs once